pkgs/os-specific/linux/kernel/generic.nix

```diff
@@ -71,7 +71,7 @@ lib.makeOverridable ({ # The kernel source tarball.
 , autoModules ? stdenv.hostPlatform.linux-kernel.autoModules
 , preferBuiltin ? stdenv.hostPlatform.linux-kernel.preferBuiltin or false
 , kernelArch ? stdenv.hostPlatform.linuxArch
-, kernelTests ? []
+, kernelTests ? {}
```

Comment:
This is technically an API break, can we maybe handle both cases for a bit?

Reply:
Do we think people are passing in custom kernel tests much?  I'd be surprised…